kubeflow/contrib/kserve/patches/config-param.yaml

```diff
@@ -1,4 +0,0 @@
-- op: replace
-  path: /data
-  value:
-    ingressGateway: kubeflow/kubeflow-gateway
```

Comment:
How is ingressGateway being set in this upgrade?

Reply:
This ConfigMap was removed as a part of cleanup in KServe 0.10. I believe this is now hardcoded in the `kubeflow_kserve.yaml`. Let me know what's your opinion on this. Thank you!

Reply:
Which file is kubeflow_kserve.yaml? Can you point me to the file? Thank you!

Reply:
I found it in https://github.com/kubeflow/manifests/blob/master/contrib/kserve/kserve/kserve_kubeflow.yaml, sounds good on removing this file then. Thank you!